src/katsdpcontroller/generator.py

```diff
@@ -2206,8 +2213,8 @@ def _make_beamformer_engineering_pol(
         Whether this node is for writing to ramdisk (ignored if `stream` is ``None``)
     idx
         Number of this source within the output (ignored if `stream` is ``None``)
-    develop
-        Whether this is a develop-mode config
+    develop_opts
+        develop_opts of this config
```

Comment:
Actually it doesn't seem to be passed at all, so just delete it from the docstring (it's retrieved from `configuration`).

Reply:
Done